Lib/_pydatetime.py

```diff
@@ -559,23 +559,24 @@ def _check_utc_offset(name, offset):
         return
     if not isinstance(offset, timedelta):
         raise TypeError("tzinfo.%s() must return None "
-                        "or timedelta, not '%s'" % (name, type(offset)))
+                        "or timedelta, not '%s'"
+                        % (name, type(offset).__name__))
```

Comment:
```suggestion
                        f"or timedelta, not {type(offset).__name__!r}"
```

Reply:
@ZeroIntensity why did you remove the quotes? this makes the error messages different

Reply:
It shouldn't, using `!r` will trigger the repr and make the quotes appear.

Reply:
Oh, sorry, forgot about it

Reply:
i already got it